packages/sdk/src/tasks/finalize-withdrawal.ts

```diff
@@ -147,8 +170,12 @@ task('finalize-withdrawal', 'Finalize a withdrawal')
       clearInterval(finalizeInterval)
     }
 
-    const tx = await messenger.finalizeMessage(txHash)
-    receipt = await tx.wait()
-    console.log(receipt)
-    console.log('Finalized withdrawal')
+    try {
+      const tx = await messenger.finalizeMessage(txHash)
+      receipt = await tx.wait()
+      console.log(receipt)
+      console.log('Finalized withdrawal')
```

Comment:
This seems to go out of `try` scope so that `Finalized withdrawal` is always printed.